### DISCUSSION TITLE
start with rpc server ?

### DISCUSSION THREAD

**Discussion by acodervic** [projectdiscovery/nuclei#2581] [Ideas]
I intend to embed nuclei in my program, but is there any similar functionality for rpcServer other than invoking nuclei from the command line?

**Comment by forgedhallpass** [comment]
@acodervic nuclei can be used as a go library. Here is an example: https://github.com/projectdiscovery/nuclei/blob/master/DESIGN.md#using-nuclei-from-go-code

Later we'll add a REST API as well: https://github.com/projectdiscovery/nuclei/issues/1276

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
